sqlite/migrations/staticcheck.conf

```diff
@@ -0,0 +1 @@
+checks = ["inherit", "-ST1005"]
```

Comment:
Can you add a comment above this line explaining what it does & why it's needed?